openfisca_us/variables/gov/irs/income/sources/social_security/tax_exempt_social_security.py

```diff
@@ -0,0 +1,14 @@
+from openfisca_us.model_api import *
+
+
+class tax_exempt_social_security(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Tax-exempt social security"
```

Comment:
Looks like the IRS is inconsistent about capitalization, but I typically see it capitalized.
```suggestion
    label = "Tax-exempt Social Security"
```